### PR TITLE
*Use .AB instead of .application

### PR DESCRIPTION
**Pull request by wongpratan**

### CODE REVIEW

**Review by Achoobert — APPROVED** [review]
LGTM

File: process/tasks/ABProcessElementCore.js [hunk 1]
```diff
@@ -455,10 +455,10 @@ module.exports = class ABProcessTaskCore extends ABMLClass {
    }
 
    get objectOfStartElement() {
-      let startElem = this.startElements[0];
+      const startElem = this.startElements[0];
       if (!startElem) return null;
 
-      let startElemObj = this.application.objects(
+      const startElemObj = this.AB.objects(
```
Comment (nh758) [inline process/tasks/ABProcessElementCore.js]: I think you can use `this.AB.objectByID(startElem.objectID)`

Comment (wongpratan) [reply process/tasks/ABProcessElementCore.js]: Thank you. It should be updated.

**Review by jamesduncan — APPROVED** [review]

**Comment by jamesduncan** [comment]
Once this is merged will the other PR https://github.com/digi-serve/appbuilder_platform_service/pull/37#pullrequestreview-1136532527 pass it's tests?

**Review by nh758 — APPROVED** [review]

**Pull request merged**